app/gateway/changelog.md

```diff
@@ -3,6 +3,493 @@ title: Kong Gateway Changelog
+
+* You can customize the transparent dynamic TLS SNI name.
+
+  Thanks, [@zhangshuaiNB](https://github.com/zhangshuaiNB)!
+  [#8196](https://github.com/Kong/kong/pull/8196)
+
+* Routes now support matching headers with regular expressions.
+
+  Thanks, [@vanhtuan0409](https://github.com/vanhtuan0409)!
+  [#6079](https://github.com/Kong/kong/pull/6079)
+
+* You can now configure [`cluster_max_payload`](/gateway/latest/reference/configuration/#cluster_max_payload)
+  for hybrid mode deployments. This configuration option sets the maximum payload
+  size allowed to be sent across from the control plane to the data plane. If your
+  environment has large configurations that generate `payload too big` errors
+  and don't get applied to the data planes, use this setting to adjust the limit.
+
+  Thanks, [@andrewgknew](https://github.com/andrewgknew)!
+  [#8337](https://github.com/Kong/kong/pull/8337)
+
+#### Performance
+
+* Improved the calculation of declarative configuration hash for big configurations.
+  The new method is faster and uses less memory.
+  [#8204](https://github.com/Kong/kong/pull/8204)
+
+* Multiple improvements in the Router, including:
+  * The router builds twice as fast
+  * Failures are cached and discarded faster (negative caching)
+  * Routes with header matching are cached
+
+  These changes should be particularly noticeable when rebuilding in DB-less
+  environments.
+  [#8087](https://github.com/Kong/kong/pull/8087)
+  [#8010](https://github.com/Kong/kong/pull/8010)
+
+#### Admin API
+
+* The current declarative configuration hash is now returned by the `status`
+  endpoint when Kong node is running in DB-less or data plane mode.
+  [#8214](https://github.com/Kong/kong/pull/8214)
+  [#8425](https://github.com/Kong/kong/pull/8425)
+
+#### Plugins
+
+* [Canary](/hub/kong-inc/canary/) (`canary`)
+  * Added the ability to configure `canary_by_header_name`. This parameter
+  accepts a header name that, when present on a request, overrides the configured canary
+  functionality.         
+    * If the configured header is present with the value `always`, the request will
+      always go to the canary upstream.
+    * If the header is present with the value `never`, the request will never go to the
+    canary upstream.
+
+* [Prometheus](/hub/kong-inc/prometheus/) (`prometheus`)
+  * Added three new metrics:
+    * `kong_db_entities_total` (gauge): total number of entities in the database.
+    * `kong_db_entity_count_errors` (counter): measures the number of errors encountered during the measurement of `kong_db_entities_total`.
+    * `kong_nginx_timers` (gauge): total number of Nginx timers, in Running or Pending state. Tracks `ngx.timer.running_count()` and
+      `ngx.timer.pending_count()`.
+      [#8387](https://github.com/Kong/kong/pull/8387)
+
+* [OpenID Connect](/hub/kong-inc/openid-connect/) (`openid-connect`)
+
+  * Add Redis ACL support (Redis v6.0.0+) for storing and retrieving a session.
```

Comment:
`  * Added Redis ACL support (Redis v6.0.0+) for storing and retrieving a session.`